consul/check.py

```diff
@@ -246,18 +246,27 @@ def check(self, instance):
             # Make service checks from health checks for all services in catalog
             health_state = self.consul_request(instance, '/v1/health/state/any')
 
+            sc = {}
+            # compute the highest status level (OK < WARNING < CRITICAL) a a check among all the nodes is running on.
             for check in health_state:
+                sc_id = '{0}-{1}-{2}'.format(check['CheckID'], check.get('ServiceID', ''), check.get('ServiceName', ''))
                 status = self.STATUS_SC.get(check['Status'])
                 if status is None:
                     continue
```

Comment:
Make sens. we should handle `AgentCheck.UNKNOWN` as the least priority for a health check. Which loop back with you next comment.